GTMAppAuth/Sources/Public/GTMAppAuth/GTMAppAuthFetcherAuthorization+Keychain.h

```diff
@@ -33,12 +33,33 @@ NS_ASSUME_NONNULL_BEGIN
 + (nullable GTMAppAuthFetcherAuthorization *)
     authorizationFromKeychainForName:(NSString *)keychainItemName;
 
+/*! @brief Attempts to create a @c GTMAppAuthFetcherAuthorization from data stored in the keychain
+        in GTMAppAuth format.
+    @param keychainItemName The keychain name.
+    @param dataProtectionKeychain Whether or not to use the data protection
+        keychain on macOS 10.15+.
+    @return A @c GTMAppAuthFetcherAuthorization object, or nil.
+ */
++ (nullable GTMAppAuthFetcherAuthorization *)
+    authorizationFromKeychainForName:(NSString *)keychainItemName
```

Comment:
Since we already have a method to do authorization without data protection. We may consider removing this boolean and name is `authorizationFromKeychainWithDataProtectionForName:(NSString *)keychainItemName`. This also prevents the clients from using this method with flag YES and NO altogether.

Reply:
I considered this route, but I'm worried it results in method signatures that are too hard to read.